modules/data_warehouse/README.md

```diff
@@ -1,26 +1,26 @@
-# terraform-google-ssw
+# terraform-google-bigquery-data-warehouse
 
 ## Description
-### tagline
```

Comment:
Let's keep these headings aligned with template as it is used in metadata generation.
https://github.com/terraform-google-modules/terraform-google-module-template/blob/master/terraform-google-%7B%7Bcookiecutter.module_name%7D%7D/README.md